.env.j2

```
@@ -0,0 +1,9 @@
+LASK_APP = todo_app/app
```

A typo when copying this over I think!
```suggestion
FLASK_APP = todo_app/app
```